inc/packages/admin/info.php

```diff
@@ -322,10 +325,38 @@ function render_fyi( MetadataDocument $doc, ReleaseDocument $release ) : void {
 				?>
 			</ul>
 		<?php endif ?>
+		<?php
+		$repo_host = get_repository_hostname( $doc->id );
+		if ( $repo_host ) :
+			?>
+			<p><small>Plugin available via FAIR repository hosted at <?php echo esc_html( $repo_host ); ?></small></p>
+		<?php else: ?>
+			<p><small>Plugin available via FAIR repository</small></p>
+		<?php endif; ?>
 	</div>
 	<?php
 }
 
+/**
+ * Get the hostname for the repository hosting a package.
+ *
+ * @param string $did DID to check.
+ * @return string|null Hostname if available, null if there's an error.
+ */
+function get_repository_hostname( string $did ) : ?string {
+	$did_doc = Packages\get_did_document( $did );
+	if ( is_wp_error( $did_doc ) ) {
+		return null;
+	}
+
+	$repo = $did_doc->get_service( Packages\SERVICE_ID );
+	if ( empty( $repo ) ) {
+		return null;
+	}
+
+	return parse_url( $repo->serviceEndpoint, PHP_URL_HOST );
```

Comment:
In the event of a malformed URL, `parse_url()` will return `false`. Should we handle this in case the DID document (and serviceEndpoint) is created by something other than Mini FAIR repo?

Reply:
I don't think it'd ever end up here (since you wouldn't be able to actually load the metadata if the DID was malformed), but can do.